libr/include/r_io.h

```diff
@@ -192,6 +191,7 @@ typedef struct {
 } RIORap;
 
 typedef struct r_io_plugin_t {
+	// RPluginMeta
```

Comment:
mark as R2_590, so we can change it in a future PR

Reply:
i did not marked it because it feels like we are breaking so much stuff and this will require patching more projects xD like r2frida, but yeah ideally i would like to have all the plugins this way.